src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.Log.cs

```diff
@@ -218,6 +230,38 @@ public static void ErrorDuringClosedEvent(ILogger logger, Exception exception)
             {
                 _errorDuringClosedEvent(logger, exception);
             }
+
+            public static void TransportNotSupported(ILogger logger, string transport)
+            {
+                if (logger.IsEnabled(LogLevel.Debug))
```

Comment:
not needed

Reply:
Yep, this used to be an object that had to be `.ToString`ed, but isn't anymore, I'll remove it.